sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling-multi-gpu-single-node.ipynb

```diff
@@ -177,7 +177,7 @@
    "id": "accepting-burden",
```

Comment:
<p>change back to "an <code>ml.p3.8xlarge</code>".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I think I get a grammar error if I do that.

Reply:
That's ok. Please change to "an"